packages/lib/CalendarService.ts

```diff
@@ -359,15 +360,24 @@ export default abstract class BaseCalendarService implements Calendar {
 
         const start = dayjs(dateFrom);
         const end = dayjs(dateTo);
-        const iterator = event.iterator();
-        let current;
+        const startDate = ICAL.Time.fromDateTimeString(startISOString);
+        startDate.hour = event.startDate.hour;
+        startDate.minute = event.startDate.minute;
+        startDate.second = event.startDate.second;
+        const iterator = event.iterator(startDate);
+        let current = iterator.next();
```

Comment:
to note this should be just 
```
let current;
```

otherwise the first entry gets skipped.

seems like this got missing.